cmd/tools_netem.go

```diff
@@ -83,6 +88,11 @@ var netemShowCmd = &cobra.Command{
 }
 
 func netemSetFn(_ *cobra.Command, _ []string) error {
+	// Ensure that the sch_netem kernel module is loaded (for Fedora/RHEL compatibility)
+	if err := exec.Command("modprobe", "sch_netem").Run(); err != nil {
```

Comment:
@FloSch62 can you maybe try to get rid of this exec and use the module loading code we have in place already?
https://github.com/srl-labs/containerlab/blob/f861ff64b0408c1bdc6ad2046512702a74610fce/clab/config.go#L393